airgun/views/report_template.py

```diff
@@ -86,3 +87,21 @@ def is_displayed(self):
             and self.breadcrumb.locations[0] == 'Report Templates'
             and self.breadcrumb.read() == 'Create Template'
         )
+
+
+class ReportTemplateGenerateView(BaseLoggedInView):
+    breadcrumb = BreadCrumb()
+    submit = Text('//input[@name="commit"]')
```

Comment:
please put widget in the order we have them in UI as they will be filled in that order
and more important, where is `Generate at` field which is present in every view

Reply:
That field will not get to the `TextInputsGroup`, right?

Reply:
I mean we created `TextInputsGroup` for dynamic content, and that one is static for every report

Reply:
The Generate at field should be filled extra, not as part of TextInputsGroup, exactly as you say. I will reorder the fields. I don't want to include the Generate At field right now because it's a datetime picker, which
1) is a new thing in the WebUI
2) is somewhat hard to implement
3) is not necessary now
4) will be reworked soon

Reply:
Sounds good to me, it is definitely large piece of work and out of scope for current PR